CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ END_UNRELEASED_TEMPLATE
 * (toolchain) Python 3.13 now references 3.13.5
 * (gazelle) Switched back to smacker/go-tree-sitter, fixing
   [#2630](https://github.com/bazel-contrib/rules_python/issues/2630)
+* (gazelle) Cleanup multi-line python imports modules
```

Comment:
Nit1: Move to "fixed" section

Nit2: The current wording makes it sound like we just did some internal refactoring. Consider something like `Multi-line python imports (eg with escaped newlines) are now correctly processed by Gazelle.`.